### PR TITLE
MONIT-17772: Implemented re-ingestion of WF traces.

### PR DESCRIPTION
**Pull request by dbaghdasarya**

### CODE REVIEW

File: src/main/java/com/wavefront/generators/ReIngestGenerator.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+//        shift = endMillis - shift;
+//      } else {
+//        shift = 0;
+//      }
+//      Thread.sleep(shift + 7_200_000);
+//    } catch (InterruptedException e) {
+//      LOGGER.severe(e.toString());
+//    }
+//    reGenerateTraces(false,
+//        new Pair<String, String>("service", "preference_service"), 80);
```
Comment (sirak92) [inline src/main/java/com/wavefront/generators/ReIngestGenerator.java]: is it intentional?

File: src/main/java/com/wavefront/generators/ReIngestGenerator.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+              }
+            } catch (IOException | InterruptedException e) {
+              LOGGER.severe(e.toString());
+            }
+          }
+        }
+      });
+
+      SimpleDateFormat simpleDateFormat = new SimpleDateFormat("HH:mm:ss");
+      String endTime = simpleDateFormat.format(new Date(endMoment.get()));
```
Comment (sirak92) [inline src/main/java/com/wavefront/generators/ReIngestGenerator.java]: endTime is not used

**Review by sirak92 — COMMENTED** [review]
As soon as it's tested and works fine, lgtm

File: src/main/java/com/wavefront/generators/ReIngestGenerator.java (hunk 1 above)
Comment (dbaghdasarya) [reply src/main/java/com/wavefront/generators/ReIngestGenerator.java]: Will remove it

File: src/main/java/com/wavefront/generators/ReIngestGenerator.java (hunk 2 above)
Comment (dbaghdasarya) [reply src/main/java/com/wavefront/generators/ReIngestGenerator.java]: Will remove it

**Pull request merged**